ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js

```diff
@@ -142,7 +142,7 @@ const EditGasToolTip = ({
               fontWeight={FontWeight.Bold}
               className="edit-gas-tooltip__container__label"
             >
-              {t('priorityFeeProperCase')}
+              {t('priorityFee')}
```

Comment:
Nit: This seems like another change that we could exclude from this PR, to reduce the diff of the hotfix and avoid conflicts

Reply:
Sorry, must have missed this one when I was undoing those changes. Pushed changes now